boxes/templates/boxes/box_list.html

```diff
@@ -111,7 +111,8 @@ <h3 class="box__title text text-darkest">{{box.name}}</h3>
     <div class="row center-xs center-md center-lg">
       <div class="col-xs-12 col-sm-12 col-md-12">
         <h3 class="subtitle center-xs">Create New Box</h3>
-        <p class="smalltext text-lightest "> For others to be able to send you encrypted content through the web page, you will need to generate "boxes". Each box has an associated link that you can copy and share with any given person, it can only be used once. After the expiration date, the form is locked and the link can no longer be used.</p>
+        <p class="smalltext text-lightest "> For others to be able to send you encrypted content through Hawkpost, you'll need to generate "boxes".</p> 
+        <p class="smalltext text-lightest ">  Each box has an associated link that you can copy and share with any given person. It can only be used once and until the expiration date. After that the form is locked and the link cannot longer be used.</p>
```

Comment:
yes, you are right that info is outdated, now we have `never expires` and `multiple messages`